app-android/src/main/java/com/ustadmobile/port/android/view/RegisterAgeRedirectFragment.kt

```diff
@@ -4,30 +4,14 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.compose.foundation.layout.*
```

Comment:
This file is not related to the screen. It should not be changed in this pull request.